pkg/api/nitric/v1/document.pb.validate.go

```diff
@@ -32,7 +31,6 @@ var (
 	_ = (*url.URL)(nil)
 	_ = (*mail.Address)(nil)
 	_ = anypb.Any{}
-	_ = sort.Sort
```

Comment:
@jyecusch can you double check the version of protoc/protoc-gen-go you're using for this? It looks like there is a mismatch with the version we're using for CI/CD. I think @asalkeld implmeneted some install scripts in our makefile for this to ensure the versions are consistent, so it looks like we'll need to double check this.

Reply:
I have 3.19.4 installed, it looks like the install script is trying to install 3.19.1